exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/diagnosiskey/DiagnosisKeyDao.java

```diff
@@ -57,6 +57,7 @@ public int deleteVerificationsBefore(Instant verificationTime) {
     public void addKeys(int verificationId, String requestChecksum, int interval, List<TemporaryExposureKey> keys) {
         if (verify(verificationId, requestChecksum) && !keys.isEmpty()) {
             batchInsert(interval, keys);
+            addDiagnosisKeyStatsRow(Instant.now());
```

Comment:
This won't be called when someone makes a valid push-request with a valid token but 0 keys (fresh installation). I think those should be included in the stat, as it will be compared to publish token creation stats. Hence, it might be better to call it from within verify(..) method. As such, it could also be named "stats_token_verified" or somesuch.

Reply:
Ok, good catch. I'll fix that

Reply:
Fixed